.gitignore

```diff
@@ -1,7 +1,6 @@
 generatedWebHelp/
 lib/
 node_modules/
-prototyping-dir/
-package-lock.json
 commandTree.json
-zowe.json
\ No newline at end of file
+zowe.json
+tsconfig.tsbuildinfo
```

Comment:
Do we want to keep package-lock in there? That might be an issue when it comes to merge conflicts.

Reply:
I think this is acceptable for our use case. Honestly, I don't think any changes in transitive dependencies will affect the output of these scripts, and the headache of merge conflicts is something we want to avoid in this repo as much as possible (the SPIKE specifically laid out that our plan needs to avoid merge conflicts).